File: sdks/sdkwork-craw-chat-sdk-admin/sdkwork-craw-chat-sdk-admin-flutter/composed/lib/src/meta_module.dart
import 'package:craw_chat_admin_backend_sdk/craw_chat_admin_backend_sdk.dart';

import 'context.dart';

class CrawChatAdminMetaModule {
  final CrawChatAdminSdkContext context;

  CrawChatAdminMetaModule(this.context);

  Future<JsonObject> health() {
    return context.backendClient.meta.getHealthz();
  }
}

File: sdks/sdkwork-craw-chat-sdk-admin/sdkwork-craw-chat-sdk-admin-typescript/composed/src/meta-module.ts
import type { JsonObject } from './types.js';
import type { CrawChatAdminSdkContext } from './sdk-context.js';

export class CrawChatAdminMetaModule {
  constructor(private readonly context: CrawChatAdminSdkContext) {}

  health(): Promise<JsonObject> {
    return this.context.backendClient.meta.getHealthz();
  }
}
